webapp-django/crashstats/signature/static/signature/css/signature_report.less

```diff
@@ -118,7 +119,19 @@ a.crash-id {
     }
 }
 
-#graph-panel {
+.graph-panel {
+    .panel {
+        .options {
```

Comment:
Why is this on ``.options`` and not on ``.delete``? It seems to be more specific to the latter one.